test/integration_tests_no_travis.sh

```diff
@@ -1,26 +1,43 @@
 #!/bin/bash
 
-set -ex
+set -e
+
+function kill_children() {
+  for job in $(jobs -p)
+  do
+    kill -s SIGINT "$job"
```

Comment:
Replace this with SIGTERM; it might improve the issues you were having with Ctrl-C hanging